account_payment_promissory_note_number/views/account_payment_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="view_account_payment_form" model="ir.ui.view">
+        <field name="name">account.payment.form</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_form"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='company_id']" position="after">
+                <field name="promissory_note_number" string="Promissory note number"/>
```

Comment:
`string` parameter is innecessary because field already has that label. Only use it if in the selected view we can use an alternative label.
```suggestion
                <field name="promissory_note_number">
```